File: angular/src/app/hotel/models/hotel-address-dto.ts

import { HotelDto } from './hotel-dto';

import { EntityDto } from '@abp/ng.core';

export class HotelAddressDto extends EntityDto<string> {
  province: string;
  district: string;
  city: string;
  hotelId: string;
  hotel: any;
  id: string;

  constructor(initialValues: Partial<HotelAddressDto> = {}) {
    super(initialValues);
  }
}

File: aspnet-core/src/HotelApp.Application.Contracts/Hotels/HotelAddressDto.cs
using System;
using Volo.Abp.Application.Dtos;

namespace HotelApp.Hotels
{
    public class HotelAddressDto : EntityDto<Guid>
    {
        public string Province { get; set; }
        public string District { get; set; }
        public string City { get; set; }

        public Guid HotelId { get; set; }

        public HotelDto Hotel { get; set; }
    }



}
